src/helics/core/helicsCLI11.hpp

```diff
@@ -142,7 +142,7 @@ class helicsCLI11App: public CLI::App {
     {
         auto* og = add_option_group("network type")->immediate_callback();
         og->add_option_function<std::string>(
-              "--coretype,-t,--type,--core",
+              "--coretype,--core_type,--coreType,-t",
```

Comment:
Can the ignore underscore and case options be used so all 3 aren't listed?

Reply:
I'm generally OK with that but the application of the ignore underscore and ignore capitalization is hit or miss throughout the code base. My thought was to have all three listed all the time so as to remove any doubt. Then, when we're ready to go back through and do housekeeping, we could remove the alternatives and add in the normalization functions universally.

But if there's a good reason to not do that now I'm fine with it. My goal was consistency and clarity as a part of the v3 transition.

Reply:
yeah thought I caught most of them but missed a few apparently